tests/plugins/targetlocker/dblocker/dblocker_test.go

```diff
@@ -23,6 +23,7 @@ import (
 
 var (
 	jobID = types.JobID(123)
+	jobRargetManagerAcquireTimeout = 5 * time.Minute
```

Comment:
The timeout for all these tests should be 2 seconds for initial + 2 seconds for refresh, see line 42.
Does this compile?

Reply:
Yep, I realised this now, changing it.

Reply:
I've fixed the lock timeout to be 2 seconds but, apparently, it doesn't really matter if the refresh timeout is left unchanged (e.g. 5 minutes from the default config).